.github/workflows/selfhosted.yml

```diff
@@ -47,13 +47,26 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: 3.9
+        python-version: "3.10"
+    - name: Install and configure poetry
+      uses: snok/install-poetry@v1
+      with:
+        virtualenvs-create: true
+        virtualenvs-in-project: true
+    - name: Disable modern installer
+      run: |
+        poetry config installer.modern-installation false
```

Comment:
Is there a specific reason for this? The issue with `furo` should have been solved, do we have any other bloody package?

Reply:
No, it is just copy/paste from https://github.com/qiboteam/workflows/blob/main/.github/workflows/rules-poetry.yml#L46.

Reply:
Ok, so we should have updated the other one first @Edoardo-Pedicillo

However, I saw you actually simplified, and the runner passed with the same amount of errors as before. So, please keep going :)

Reply:
> However, I saw you actually simplified, and the runner passed with the same amount of errors as before. So, please keep going :)

@AleCandido if I had time that would be very fun, indeed ;-).